custom/enikshay/case_utils.py

```diff
@@ -585,9 +585,13 @@ def get_all_episode_ids(domain):
     return case_ids
 
 
-def iter_all_active_person_episode_cases(domain, case_ids):
+def iter_all_active_person_episode_cases(domain, case_ids, sector="both"):
     """From a list of case_ids, return all the active episodes and associate person case
     """
+    accepted_sectors = ['both', 'private', 'public']
```

Comment:
I usually try to use a tuple or a constant with something like this so that it can't accidentally be modified later